src/osp/Resource/Package.h

```diff
@@ -61,12 +61,26 @@ namespace osp
 
 // supported resources
 
+// Just a string, but typedef'd to indicate that it represents a prefix
+typedef std::string resprefix_t;
```

Comment:
Use the using keyword over typedef. Typedef sucks.

Reply:
The primary reason why I did that was so that `OSPApplication::m_packages` was a `std::map<resprefix_t, Package>` which makes it more obvious that the key is not just any old string but rather a specific engine construct. Sort of like how `size_t` (which is just `uint64_t`) conveys a specific meaning. I'll leave it up to you whether or not I should leave it this way.